README.md

```diff
@@ -13,14 +13,13 @@ Download the latest `acli.phar` file from the [releases](https://github.com/acqu
 
 For example, you can run following simple script:
 ```bash
-curl -OL https://github.com/acquia/cli/releases/download/v1.0.0-rc3/acli.phar
-mv acli.phar acli
-chmod +x acli
+curl -OL https://github.com/acquia/cli/releases/download/v1.0.0-rc4/acli.phar
```

Comment:
Maybe would be better to point to /latest, but could not find for some reason the right link

Reply:
The problem is not linking to the latest release, it's finding how to [link to the correct PHAR file](https://gist.github.com/steinwaywhw/a4cd19cda655b8249d908261a62687f8). And that requires a pretty advanced one-liner so I'm not sure we can afford doing that for newcomers to Acquia CLI.

Perhaps that should be part of the release runbook to modify the README file.

Reply:
I’m pretty sure that the link for the latest asset doesn’t work because we don’t have a stable release yet. Once we have a stable release, it should start working and we can update this file. Until then, let’s just incrementally update the file manually.

Reply:
that thought of no stable release came to my mind